solitude/settings/base.py

```diff
@@ -349,15 +349,22 @@
     '--with-blockage',
 ]
 
-if not os.environ.get('LIVE_TESTS'):
-    NOSE_ARGS += [
-        '-a !live',
-        '--http-whitelist=']
+# Figure out which tests we run. By default that's no live tests.
+# LIVE_TESTS=live,braintree  < runs tests marked as live and braintree
+tests = os.environ.get('LIVE_TESTS', '')
+
+if not tests:
+    # If you specified no tests, exclude them all by default.
+    NOSE_ARGS.append('-a !{0}'.format(',!'.join(['live', 'braintree'])))
 else:
-    NOSE_ARGS += [
-        '-a live',
-        '--http-whitelist=localhost,api.sandbox.braintreegateway.com'
-    ]
+    # If you specified something, just pass it through.
+    NOSE_ARGS.append('-a ' + tests)
+
+# Figure out the whitelist.
+http = 'localhost' if 'live' in tests else ''
+http += ',api.sandbox.braintreegateway.com' if 'braintree' in tests else ''
```

Comment:
this looks like you could end up with ``--http-whitelist=,api.sandbox.braintreegateway.com`` (a preceding comma). That seems like it could break something. You could work around it by making ``http`` a list and doing ``",".join(http)``